CMakeLists.txt

```diff
@@ -51,9 +51,9 @@ configure_file(
     "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
     IMMEDIATE @ONLY)
 
-add_custom_target(uninstall
+add_custom_target(uninstall-Vulkan-Tools
```

Comment:
I think a more permanent and better solution is possible:

How about checking to see if the uninstall target is defined and if not, run all this code to define it.

That way, this repo can be included as a submodule by another repo that defines uninstall and only the top level repo defines uninstall, which is what you want.  Further, you don't have to invent odd "uninstall-repo-name" target names.

Reply:
Ah, nice!  I'll give it a try. Do you recommend doing the same in other repos with uninstall targets, or just here?

Reply:
We should do it to all 4 of the new repos and any others that we have that might be used as submodules.   I don't know the exact CMake syntax for checking this, but `if(NOT uninstall)` would be a good start.

Reply:
if(NOT TARGET uninstall)